build fails without git in $PATH

Building powsybl-tools in tools/ without ```git``` in $PATH:
```
[...]
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ powsybl-tools ---
[INFO] Executing: /bin/sh -c cd '/tmp/powsybl-core-master/tools' && 'git' 'rev-parse' '--verify' 'HEAD'
[INFO] Working directory: /tmp/powsybl-core-master/tools
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] powsybl ............................................ SUCCESS [  1.423 s]
[INFO] Commons ............................................ SUCCESS [  0.837 s]
[INFO] Computation API .................................... SUCCESS [  0.229 s]
[INFO] Tools .............................................. FAILURE [  0.410 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 3.379 s
[INFO] Finished at: 2019-02-04T13:56:11+01:00
[INFO] Final Memory: 38M/584M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:buildnumber-maven-plugin:1.4:create (default) on project powsybl-tools: Cannot get the revision information from the scm repository : 
[ERROR] Exception while executing SCM command.: Error while executing command. Error while executing process. Cannot run program "git" (in directory "/tmp/powsybl-core-master/tools"): error=2, Aucun fichier ou dossier de ce type
[ERROR] -> [Help 1]
```


Note that if the source code we are compiling is not a git checkout (for example using "download zip" from github), it does work if git is in the path, detecting UNKNOWN_BRANCH:
```
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ powsybl-tools ---
[INFO] Executing: /bin/sh -c cd '/tmp/powsybl-core-master/tools' && 'git' 'rev-parse' '--verify' 'HEAD'
[INFO] Working directory: /tmp/powsybl-core-master/tools
[INFO] Storing buildNumber: null at timestamp: 1549285149505
[WARNING] Cannot get the branch information from the git repository: 
Detecting the current branch failed: fatal: Not a git repository (or any parent up to mount point /)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

[INFO] Executing: /bin/sh -c cd '/tmp/powsybl-core-master/tools' && 'git' 'rev-parse' '--verify' 'HEAD'
[INFO] Working directory: /tmp/powsybl-core-master/tools
[INFO] Storing buildScmBranch: UNKNOWN_BRANCH
[INFO] 
```

It should work without git IMHO

--- a/pom.xml
+++ b/pom.xml
@@ -306,6 +306,7 @@
                     <configuration>
                         <doCheck>false</doCheck>
                         <doUpdate>false</doUpdate>
+                        <revisionOnScmFailure>UNKNOWN_BUILD</revisionOnScmFailure>
                     </configuration>
                 </plugin>
                 <plugin>

